gBuilder.cm* no longer distributed in beta4

Hi,

gBuilder.ml* is no longer compiled in beta4. Therefore the code generated by lablgladecc does not
work either.

Sorry for not having check before, I was on mission and I am just back.

@ejgallego 

The .o library for ml_gtkbuilder.c is missing.

/usr/bin/ld: /home/claudio/.opam/4.07.1/lib/lablgtk3/lablgtk3.a(gtkBuilderProps.o): in function `camlGtkBuilderProps__entry':
/home/claudio/.opam/4.07.1/.opam-switch/build/lablgtk3.3.0.beta4/_build/default/src/gtkBuilderProps.ml:5: undefined reference to `ml_gtkbuilder_init'
/usr/bin/ld: /home/claudio/.opam/4.07.1/lib/lablgtk3/lablgtk3.a(gtkBuilderProps.o): in function `camlGtkBuilderProps__6':
:(.data+0x100): undefined reference to `ml_gtk_builder_get_object'
/usr/bin/ld: :(.data+0x108): undefined reference to `ml_gtk_builder_add_objects_from_string'
/usr/bin/ld: :(.data+0x110): undefined reference to `ml_gtk_builder_add_objects_from_file'

--- a/examples/glade/dune
+++ b/examples/glade/dune
@@ -1,0 +1,19 @@
+(executables
+ (names glade_demo)
+ (modules project1 glade_demo)
+ (flags :standard -no-strict-sequence -warn-error -10)
+ (libraries lablgtk3))
+
+(executables
+ (names gladecalc)
+ (modules project2 gladecalc)
+ (flags :standard -no-strict-sequence -warn-error -10)
+ (libraries lablgtk3))
+
+(rule
+ (targets project1.ml)
+ (action (with-stdout-to %{targets} (run lablgladecc3 project1.ui))))
+
+(rule
+ (targets project2.ml)
+ (action (with-stdout-to %{targets} (run lablgladecc3 project2.ui))))

--- a/examples/glade/glade_demo.ml
+++ b/examples/glade/glade_demo.ml
@@ -11,7 +11,9 @@
 (* An experiment on using libglade in lablgtk *)
 
 (* lablgladecc3 project1.ui > project1.ml *)
-#use "project1.ml";;
+(* #use "project1.ml";; *)
+
+open Project1
 
 let load_file f =
   let ic = open_in f in

--- a/examples/glade/gladecalc.ml
+++ b/examples/glade/gladecalc.ml
@@ -11,7 +11,8 @@
 open StdLabels
 
 (* lablgladecc3 project2.ui > project2.ml *)
-#use "project2.ml";;
+(* #use "project2.ml";; *)
+open Project2
 
 let w1 = new window1 ()
 

--- a/src/dune
+++ b/src/dune
@@ -111,6 +111,7 @@
    ml_gdk ml_gdkpixbuf
    ml_gtk ml_gtkmisc
 
+   ml_gtkbuilder
    ml_gtkaction ml_gtkbin ml_gtkbutton ml_gtktext ml_gtkedit
    ml_gtkmenu ml_gtkfile ml_gtktree ml_gtkpack
    ; ml_gtkbroken ml_gtklist

